www/js/splash/startprefs.js

```diff
@@ -230,8 +230,8 @@ angular.module('emission.splash.startprefs', ['emission.plugin.logger',
       return startprefs.getPendingOnboardingState().then(function(result){
         if (result == null) {
           var temp = ReferralHandler.getReferralNavigation();
-          if (temp == 'recommendations') {
-            return 'root.main.recommendations';
+          if (temp == 'goals') {
```

Comment:
@trevor-wu This revert is not complete. The file still has the other extraneous changes, and they *will* cause merge conflicts. Please use git to revert the changes instead of reverting manually. This pull request is now really messy - you may also want to create a new branch, include only the relevant changes and create a new pull request from it.